### ISSUE TITLE
Endless 'Fetching contributors… Cannot retrieve contributors at this time'

### ISSUE THREAD

**Issue by grahamperrin** [sindresorhus/refined-github#1125]
Comparable to #1124 

Affected: https://github.com/captnfab/FireTitle/blob/master/README.md

With this issue, it was not enough to disable the extension then reload the page. The page appeared OK only after a quit and (re)start of Waterfox.

**Comment by bfred-it** [comment]
Can't reproduce either bug on Waterfox 56.0.4 / macOS

> The page appeared OK only after a quit and (re)start of Waterfox.

That should be proof that it's not related to Refined GitHub.

Do you see any errors in the console or network tab?

**Comment by grahamperrin** [comment]
Thanks. This one's more thought-provoking than 1124. 

Not reproducible after re-enabling the extension and reloading the page. I'll quit now, review later. 

> … in the console …

The _web_ console, yes? (Probably a daft question, but I'm only vaguely familiar with dev tools.)

----

In any case: if I find something conclusive – an extension conflict, maybe – I'll add another comment.

**Comment by bfred-it** [comment]
Yes, the browser's developer tools "Console" tab

**Comment by grahamperrin** [comment]
Just FYI as follow-up, https://www.reddit.com/r/waterfox/comments/83i3ef/-/dviyniu/ 

tl;dr

> I reloaded repeatedly, including avoidance of cached content, the problem persisted.
> 
> In a different Waterfox profile, which was created for test purposes, I found the page problem-free.
> 
After once finding the problem free in the test profile, I brought forward the problem window in my everyday profile then:
> 
> -after one simple reload, the problem ended.
> 
> Screen recording: https://photos.app.goo.gl/EGbVJNS5sm5XmQeS2 – weird, isn't it?

**Comment by bfred-it** [comment]
Thank you!

**Comment by v6** [comment]
Fixed this by disabling my Adblock on Github.
